template/project.yml

```diff
@@ -58,6 +58,11 @@ target-hooks:
     dir: ${docroot}/profiles/contrib/lightning
     command: npm run install-libraries
 
+# Configuration management options, such as core CM or features.
+# cm:
```

Comment:
I'm a little hesitant to introduce a new top level array into this file, unless we really think we're going to build out other options.

What if we just made the assumption that we'd use the machine name?